Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/50cal.yml

```
@@ -0,0 +1,77 @@
+# .50 Cal Base
+
+- type: entity
+  id: 50BulletBase
+  parent: BaseBulletTrigger
+  abstract: true
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)